src/shared/ui/SideMenu/SideMenu.tsx

```
@@ -0,0 +1,93 @@
+import { useState, useEffect } from 'react';
+import { Layout, Menu, Typography } from 'antd';
+import { LogoutOutlined } from '@ant-design/icons';
+import { useAuthStore } from '@/entities/auth/store/authStore';
+import { useLayoutStore } from '@/shared/store/layoutStore';
+import { defaultMenuItems } from '@/shared/model/menuItems';
+import { useNavigate, useLocation } from 'react-router-dom';
+import { AdminLogo } from '@/shared/ui/icons';
+import * as styles from './SideMenu.css';
+
+const { Sider } = Layout;
+const { Text } = Typography;
```

ESLint

[error] 'Text' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)